test-cypress/steps/setup/acl-management-steps.js

```diff
@@ -165,6 +169,22 @@ export class AclManagementSteps {
         return cy.get('div.small');
     }
 
+    static getPrefixWarning() {
+        return cy.get('.prefix-warning-text');
```

Comment:
Why you get stuff without context but directly from the page?! Don't do that! Try to narrow down your selectors as much as possible. Look at how the other methods in the steps are structured!